### PR TITLE
minor: remove token from ignore list that are present in google config

### PR DESCRIPTION
**Pull request by romani**
detected at https://github.com/checkstyle/checkstyle/pull/13991#discussion_r1456759351

to be merged after https://github.com/checkstyle/checkstyle/pull/13991

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java [hunk 1]
```diff
@@ -234,9 +234,12 @@ public class AllChecksTest extends AbstractModuleTestSupport {
                 "GENERIC_START", "GENERIC_END", "WILDCARD_TYPE")
                 .collect(Collectors.toUnmodifiableSet()));
         GOOGLE_TOKENS_IN_CONFIG_TO_IGNORE.put("IllegalTokenText", Stream.of(
-                // all other java tokens and text are allowed
+                // non text literals are ignored
                 "NUM_DOUBLE", "NUM_FLOAT", "NUM_INT", "NUM_LONG", "IDENT",
```
Comment (nrmancuso) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java:238]:
```suggestion
                // numerical types should not be included
                "NUM_DOUBLE", "NUM_FLOAT", "NUM_INT", "NUM_LONG",
                // identitifiers are covered by other checks
                "IDENT",
```

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java [hunk 2]
```diff
@@ -234,9 +234,12 @@ public class AllChecksTest extends AbstractModuleTestSupport {
...
-                "COMMENT_CONTENT", "STRING_LITERAL", "CHAR_LITERAL", "TEXT_BLOCK_CONTENT")
+                // comments should be skipped as nobody write in octal or unicode code style
+                "COMMENT_CONTENT",
+                // until #14291
+                "TEXT_BLOCK_CONTENT")
```
Comment (nrmancuso) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java:242]: Please wrap `)` to reduce future diff.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Items:

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java (hunk 2 above)
Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java]: done

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java (hunk 1 above)
Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java]: done

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**